firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -104,27 +130,37 @@ void homeAxis(uint8_t motor[])
     *currentMotorPos += eighthStepsPerPulse;
   }
 
-  // Moves motor towards home for fine-tuned home position
-  digitalWrite(motor[DIR_PIN], LEFT);
+  // Moves motor towards max or 0 for fine-tuned alignment
+  if (alignmentCode == POS_EXTREME)
+  {
+    digitalWrite(motor[DIR_PIN], RIGHT);
+    eighthStepsPerPulse = POS_EIGHTH_STEPS_PER_WHOLE_STEP;
+    endstopPin = MAX_ENDSTOP_PIN;
```

Comment:
Here and below (line 144), `endstopPin` already has this value; you didn't change it so don't need to set it again.